test/plugin/scenarios/druid-1.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,178 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+- serviceName: druid-1.x-scenario
+  segmentSize: ge 0
+  segments:
+  - segmentId: not null
+    spans:
+    - {operationName: Druid/Connection/getConnection, operationId: 0, parentSpanId: 1,
+      spanId: 2, spanLayer: Unknown, startTime: gt 0, endTime: gt 0,
+      componentId: 115, isError: false, spanType: Local, peer: '', skipAnalysis: false}
+    - {operationName: Druid/Connection/getConnection, operationId: 0, parentSpanId: 0,
+      spanId: 1, spanLayer: Unknown, startTime: gt 0, endTime: gt 0,
+      componentId: 115, isError: false, spanType: Local, peer: '', skipAnalysis: false}
```

Comment:
Please format YAML. Don't make mutiple components sharing one line.

Reply:
OK, I will format the expectedData.yaml.